src/Exception/InvalidRevisionException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Exception;
+
+use RuntimeException;
+
+class InvalidRevisionException extends RuntimeException
+{
+}
```

Comment:
It was meant to cover this, however it could be covered by the `notfoundexception`.
```
// and rev is invalid.
CurrentRevision() (rev Revision, err error)
```